<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRWindow</class>
 <widget class="QMainWindow" name="QRWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap>../pages_img_2025/슬라이드11.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="qr_label">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>160</y>
      <width>580</width>
      <height>580</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="print_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>830</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:1000px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>940</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:1000px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
